[BUG] Golang Install

**Describe the bug**
The Golang install with `go version go1.14.4 linux/amd64` produces too many errors to install.

**To Reproduce**
On Ubuntu 18.04 LTS I've done the following to install / setup golang, gotty, and gotty client.

```bash
# Download latest from https://golang.org/dl/
wget -c https://golang.org/dl/go1.14.4.linux-amd64.tar.gz

# Check sum
sha256sum go1.14.4.linux-amd64.tar.gz | grep aed845e4185a0b2a3c3d5e1d0a35491702c55889192bb9c30e67a3de6849c067

# Extract to /usr/local
sudo tar -C /usr/local/ -xvzf go1.14.4.linux-amd64.tar.gz

# Set environment variables
export GOPATH="$HOME/go"
export GOBIN="$GOPATH/bin"
export PATH=$PATH:/usr/local/go/bin:$GOBIN

# Install GoTTy (works)
go get github.com/yudai/gotty

# Install GoTTy-client (fails)
go get github.com/moul/gotty-client/cmd/gotty-client
```

The output I get from trying to install `gotty-client` is as follows.

```console
(base) cmccarth@luigi:~$ go get github.com/moul/gotty-client/cmd/gotty-client
# github.com/moul/gotty-client/cmd/gotty-client
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:21:5: app.Author undefined (type *cli.App has no field or method Author)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:22:5: app.Email undefined (type *cli.App has no field or method Email)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:40:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:43:4: unknown field 'EnvVar' in struct literal of type cli.BoolFlag
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:45:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:48:4: unknown field 'EnvVar' in struct literal of type cli.BoolFlag
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:50:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:53:4: unknown field 'EnvVar' in struct literal of type cli.BoolFlag
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:55:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:63:4: unknown field 'EnvVar' in struct literal of type cli.BoolFlag
go/src/github.com/moul/gotty-client/cmd/gotty-client/main.go:63:4: too many errors
```

I'm not very familiar with Go but I assumed if I had `gotty` running correctly then I could simply install `gotty-client` without any additional setup, using the command you have in the README.

Thank you for your help!

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -1,0 +1,74 @@
+env:
+  - GO111MODULE=on
+  - GOPROXY=https://proxy.golang.org
+before:
+  hooks:
+    - go mod download
+builds:
+  -
+    main: ./cmd/gotty-client
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      -
+        goos: darwin
+        goarch: 386
+    flags:
+      - "-a"
+    ldflags:
+      - '-extldflags "-static"'
+checksum:
+  name_template: '{{.ProjectName}}_checksums.txt'
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^test:'
+    - Merge pull request
+    - Merge branch
+archives:
+  -
+    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+    format_overrides:
+    -
+      goos: windows
+      format: zip
+    wrap_in_directory: true
+brews:
+  -
+    name: gotty-client
+#    github:
+#      owner: moul
+#      name: homebrew-moul
+    commit_author:
+      name: moul-bot
+      email: "bot@moul.io"
+    homepage: https://github.com/moul/gotty-client
+    description: "gotty-client"
+nfpms:
+  -
+    file_name_template: '{{ .ProjectName }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    homepage:  https://github.com/moul/gotty-client
+    description: "gotty-client"
+    maintainer: "Manfred Touron <https://manfred.life>"
+    license: "Apache-2.0 OR MIT"
+    vendor: moul
+    formats:
+      - deb
+      - rpm

--- a/README.md
+++ b/README.md
@@ -12,13 +12,13 @@
                                                               +----------------+
                +--------------+                +---------+--->|   /bin/bash    |
                |              |                |         |    +----------------+
-           +-->|   Browser    |--+             |         |                      
-+-------+  |   |              |  |             |         |                      
+           +-->|   Browser    |--+             |         |
++-------+  |   |              |  |             |         |
 |       |  |   +--------------+  |             |         |    +----------------+
 |  Bob  |--+                     +-websocket-->|  Gotty  |--->| emacs /var/www |
 |       |  |    XXXXXXXXXXXXXX   |             |         |    +----------------+
-+-------+  |   X              X  |             |         |                      
-           +-->X gotty-client X--+             |         |                      
++-------+  |   X              X  |             |         |
+           +-->X gotty-client X--+             |         |
                X              X                |         |    +----------------+
                 XXXXXXXXXXXXXX                 +---------+--->|  tmux attach   |
                                                               +----------------+
@@ -83,11 +83,15 @@ GLOBAL OPTIONS:
 
 ## Install
 
-Install latest version using Golang (recommended)
+Install latest version using Golang (recommended):
 
-```console
-$ go get github.com/moul/gotty-client/cmd/gotty-client
+```bash
+git clone https://github.com/moul/gotty-client
+cd gotty-client
+make install # or `go install ./cmd/gotty-client`
 ```
+
+_PS: I don't know why, but `go get github.com/moul/gotty-client/cmd/gotty-client` is not stable everywhere_
 
 ---
 

